README.md

```diff
@@ -300,6 +300,114 @@ config.disabledTools = disabledTools;
 config.customHeaders = {'headerName': 'headerValue'};
 PdftronFlutter.openDocument(_document, config: config);
 ```
+### PdftronFlutter.importAnnotations(String)
+Imports XFDF string to current document.
+
+```dart
+
+var xfdfCommand = '<?xml version="1.0" encoding="UTF-8"?>\n<xfdf xmlns="http://ns.adobe.com/xfdf/" xml:space="preserve">\n\t<annots>\n\t\t<circle style="solid" width="5" color="#E44234" opacity="1" creationdate="D:20190729202215Z" flags="print" date="D:20190729202215Z" page="0" rect="138.824,653.226,236.28,725.159" title="" /></annots>\n\t<pages>\n\t\t<defmtx matrix="1.333333,0.000000,0.000000,-1.333333,0.000000,1056.000000" />\n\t</pages>\n\t<pdf-info version="2" xmlns="http://www.pdftron.com/pdfinfo" />\n</xfdf>';
```

Comment:
rename to xfdf, this is not a command, a command must have at least one of <add><modify><delete>